packages/client/components/ActivityLibrary/ActivityDetails/TemplateDetails.tsx

```diff
@@ -0,0 +1,298 @@
+      <div className='flex gap-2'>
+        <DetailsBadge
+          className={clsx(CATEGORY_THEMES[category as CategoryID].primary, 'text-white')}
+        >
+          {CATEGORY_ID_TO_NAME[category as CategoryID]}
+        </DetailsBadge>
+        {!template.isFree &&
+          (lowestScope === 'PUBLIC' ? (
+            <DetailsBadge className='bg-gold-300 text-grape-700'>Premium</DetailsBadge>
+          ) : (
+            <DetailsBadge className='bg-grape-700 text-white'>Custom</DetailsBadge>
+          ))}
+      </div>
+      <div className='w-[480px]'>
+        <div className='mb-8'>
+          {isOwner ? (
+            <div className='flex items-center justify-between'>
+              <div
+                className={clsx(
+                  'w-max',
+                  isEditing && 'rounded-full border border-solid border-slate-400 pl-3'
+                )}
+              >
+                <UnstyledTemplateSharing
+                  noModal={true}
+                  isOwner={isOwner}
+                  template={template}
+                  readOnly={!isEditing}
+                />
+              </div>
+              <div className='flex gap-2'>
+                {isEditing ? (
+                  <div className='rounded-full border border-solid border-slate-400'>
+                    <DetailAction
+                      icon={'delete'}
+                      tooltip={'Delete template'}
+                      onClick={removeTemplate}
+                    />
+                  </div>
+                ) : (
+                  <>
+                    <div className='rounded-full border border-solid border-slate-400'>
+                      <DetailAction
+                        icon={'edit'}
+                        tooltip={'Edit template'}
+                        onClick={() => setIsEditing(true)}
+                      />
+                    </div>
+                    <div className='rounded-full border border-solid border-slate-400'>
+                      <CloneTemplate canClone={true} onClick={toggleTeamPickerPortal} />
+                    </div>
+                  </>
+                )}
+              </div>
+            </div>
+          ) : (
+            <div className='flex items-center justify-between'>
+              <div className='py-2 text-sm font-semibold text-slate-600'>{description}</div>
+              <div className='rounded-full border border-solid border-slate-400 text-slate-600'>
+                <FlatButton
+                  style={{padding: '8px 12px', border: '0'}}
+                  className='flex gap-1 px-12'
+                  onClick={toggleTeamPickerPortal}
+                >
+                  <ContentCopy className='text-slate-600' />
+                  <div className='font-semibold text-slate-700'>Clone & Edit</div>
+                </FlatButton>
+              </div>
+            </div>
+          )}
+        </div>
+        {ACTIVITY_TYPE_DATA_LOOKUP.description[template.type]}
```

Comment:
+1 This should really be using the `activity.description` returned by `useActivityDetails`, right?